README.md

```diff
@@ -10,6 +10,9 @@ To run this code, your Python installation must have the following modules insta
 - Pandas
```

Comment:
are Numpy etc things that get installed using `pip`, or it is anoter step. If another step, put a bulletpoint or two for each . If `pip`, remove the list as it is in `requirements.txt`.

Reply:
fixed